### PR TITLE
Describe the drag column

### PR DESCRIPTION
**Pull request by AlexJameson**

### CODE REVIEW

File: concepts/05 Widgets/DataGrid/15 Columns/10 Column Types/4 Command Columns/00 Command Columns.md [hunk 1]
```diff
@@ -18,4 +18,7 @@ Contains buttons that perform custom actions. See [Create a Column with Custom B
 - **Editing column**        
 A type of buttons column. Contains editing controls. See [Customize the Editing Column](/concepts/05%20Widgets/DataGrid/15%20Columns/10%20Column%20Types/4%20Command%20Columns/15%20Customize%20the%20Editing%20Column/0%20Customize%20the%20Editing%20Column.md '/Documentation/Guide/Widgets/DataGrid/Columns/Column_Types/Command_Columns/#Customize_the_Editing_Column').
 
+- **Drag Column**       
+Contains drag icons. Appears when **column**.[type](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/columns/#type) is "drag" and **rowDragging**.[allowReordering](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/rowDragging/#allowReordering) and [showDragIcons](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/rowDragging/#showDragIcons) are **true**.
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/DataGrid/15 Columns/10 Column Types/4 Command Columns/00 Command Columns.md]:
```suggestion
Contains drag icons. Appears when a column's [type](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/columns/#type) is *"drag"* and **rowDragging**.[allowReordering](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/rowDragging/#allowReordering) and [showDragIcons](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/rowDragging/#showDragIcons) are **true**.
```

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]
[TreeList](https://js.devexpress.com/Documentation/Guide/Widgets/TreeList/Columns/Column_Types/Command_Columns/) needs the same changes.

File: concepts/05 Widgets/DataGrid/15 Columns/10 Column Types/4 Command Columns/00 Command Columns.md [hunk 2]
```diff
@@ -18,4 +18,7 @@ Contains buttons that perform custom actions. See [Create a Column with Custom B
 - **Editing column**        
 A type of buttons column. Contains editing controls. See [Customize the Editing Column](/concepts/05%20Widgets/DataGrid/15%20Columns/10%20Column%20Types/4%20Command%20Columns/15%20Customize%20the%20Editing%20Column/0%20Customize%20the%20Editing%20Column.md '/Documentation/Guide/Widgets/DataGrid/Columns/Column_Types/Command_Columns/#Customize_the_Editing_Column').
```
Comment (arminal) [inline concepts/05 Widgets/DataGrid/15 Columns/10 Column Types/4 Command Columns/00 Command Columns.md]: A type of buttons column. --> This phrasing doesn't make sense. Can you please explain what you are trying to say? Are you saying that this is a column of different button types?

File: concepts/05 Widgets/DataGrid/15 Columns/10 Column Types/4 Command Columns/00 Command Columns.md [hunk 3]
```diff
@@ -18,4 +18,7 @@ Contains buttons that perform custom actions. See [Create a Column with Custom B
...
 
+- **Drag Column**       
+Contains drag icons. Appears when a column's [type](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/columns/#type) is *"drag"* and **rowDragging**.[allowReordering](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/rowDragging/#allowReordering) and [showDragIcons](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/rowDragging/#showDragIcons) are **true**.
```
Comment (arminal) [inline concepts/05 Widgets/DataGrid/15 Columns/10 Column Types/4 Command Columns/00 Command Columns.md]:
```suggestion
Contains drag icons. Appears when a column's [type](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/columns/#type) is *"drag"* and **rowDragging**.[allowReordering](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/rowDragging/#allowReordering) and [showDragIcons](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/rowDragging/#showDragIcons) are **true**.
```
There is a comma missing in this sentence. There are 3 "and"s and no comma. Please clarify.

File: concepts/05 Widgets/TreeList/10 Columns/10 Column Types/4 Command Columns/00 Command Columns.md [hunk 4]
```diff
@@ -9,4 +9,7 @@ Contains buttons that perform custom actions. See [Create a Column with Custom B
 - **Editing column**        
 A type of buttons column. Contains editing controls. See [Customize the Editing Column](/concepts/05%20Widgets/TreeList/10%20Columns/10%20Column%20Types/4%20Command%20Columns/15%20Customize%20the%20Editing%20Column/0%20Customize%20the%20Editing%20Column.md '/Documentation/Guide/Widgets/TreeList/Columns/Column_Types/Command_Columns/#Customize_the_Editing_Column').
```
Comment (arminal) [inline concepts/05 Widgets/TreeList/10 Columns/10 Column Types/4 Command Columns/00 Command Columns.md]: Same note as before about the first sentence's construction.

File: concepts/05 Widgets/TreeList/10 Columns/10 Column Types/4 Command Columns/00 Command Columns.md [hunk 5]
```diff
@@ -9,4 +9,7 @@ Contains buttons that perform custom actions. See [Create a Column with Custom B
...
 
+- **Drag Column**       
+Contains drag icons. Appears when a column's [type](/Documentation/ApiReference/UI_Widgets/dxTreeList/Configuration/columns/#type) is *"drag"* and **rowDragging**.[allowReordering](/Documentation/ApiReference/UI_Widgets/dxTreeList/Configuration/rowDragging/#allowReordering) and [showDragIcons](/Documentation/ApiReference/UI_Widgets/dxTreeList/Configuration/rowDragging/#showDragIcons) are **true**.
```
Comment (arminal) [inline concepts/05 Widgets/TreeList/10 Columns/10 Column Types/4 Command Columns/00 Command Columns.md]: Same note about a missing comma here.

File: concepts/05 Widgets/DataGrid/15 Columns/10 Column Types/4 Command Columns/00 Command Columns.md (hunk 2 above)
Comment (AlexJameson) [reply concepts/05 Widgets/DataGrid/15 Columns/10 Column Types/4 Command Columns/00 Command Columns.md]: That's not me (my changes are marked with green). This is an old text written long ago.

The point is the following: Editing columns make a subset of buttons columns. (Many button types are available, including `add`, `delete`, and `edit` that are used in the editing column ).

File: concepts/05 Widgets/DataGrid/15 Columns/10 Column Types/4 Command Columns/00 Command Columns.md (hunk 3 above)
Comment (AlexJameson) [reply concepts/05 Widgets/DataGrid/15 Columns/10 Column Types/4 Command Columns/00 Command Columns.md]: I'm not sure where is the best place for a comma. There are two conditions:
1. User should set the *"drag"* column's type.
1. User should assign **true ** to the `allowReordering` and `showDragIcons` options of the `rowDragging` object.

Is the following sentence more clear? 
'Appears when a column's [type]() is *"drag"*, and the [allowReordering]() and [showDragIcons]() options of the [rowDragging]() object are **true**.'

Comment (arminal) [reply concepts/05 Widgets/DataGrid/15 Columns/10 Column Types/4 Command Columns/00 Command Columns.md]: Yes, great rewrite.

**Review by RomanTsukanov — APPROVED** [review]

**Pull request merged**